.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,14 @@
-<!-- *Before creating an issue please make sure you are using the latest version of Jest, try re-installing your node_modules folder and run Jest once with `--no-cache` to see if that fixes the problem you are experiencing.* -->
+<!-- 
+THIS IS NOT A HELP FORUM. 
+If you are experiencing problems with setting up Jest, please make sure to visit our Support page: 
+https://facebook.github.io/jest/support.html
```

Comment:
We are planning on launching the new website today. Wasn't this renamed to `help.html`?